python_modules/dagster/dagster/core/launcher/launcher_daemon.py

```diff
@@ -0,0 +1,69 @@
+import os
+import subprocess
+
+from dagster import EventMetadataEntry, check, seven
+from dagster.core.events import EngineEventData
+from dagster.core.instance import DagsterInstance
+from dagster.core.storage.pipeline_run import PipelineRun
+from dagster.serdes import ConfigurableClass, ConfigurableClassData
+
+from .launcher_base import RunLauncher
+
+
+class DaemonRunLauncher(RunLauncher, ConfigurableClass):
```

Comment:
`DaemonRunLauncher` makes me think the responsibility for executing the pipeline is being handed off to some long lived daemon process - the current implementation is more so just 'fire and forget' a background process.